### ISSUE TITLE
Missing dependency on six

### ISSUE THREAD

**Issue by tnaka** [cheshirekow/cmake_format#159]
I got this error after running `pip install cmake-format` on a new machine. It seems that a dependency on six is missing.

```
$ cmake-format                                                                                                                                                                                                                                                                                                               
Traceback (most recent call last):                                                                                                                                                                                                                                                                                                              
  File "/usr/local/bin/cmake-format", line 6, in <module>                                                                                                                                                                                                                                                                                       
    from cmake_format.__main__ import main                                                                                                                                                                                                                                                                                                      
  File "/usr/local/lib/python3.7/site-packages/cmake_format/__main__.py", line 33, in <module>                                                                                                                                                                                                                                                  
    from cmake_format import configuration                                                                                                                                                                                                                                                                                                      
  File "/usr/local/lib/python3.7/site-packages/cmake_format/configuration.py", line 7, in <module>                                                                                                                                                                                                                                              
    from cmake_format.config_util import (                                                                                                                                                                                                                                                                                                      
  File "/usr/local/lib/python3.7/site-packages/cmake_format/config_util.py", line 9, in <module>                                                                                                                                                                                                                                                
    import six                                                                                                                                                                                                                                                                                                                                  
ModuleNotFoundError: No module named 'six'
```

I'm not confident but adding six in this line may fix it.
https://github.com/cheshirekow/cmake_format/blob/master/cmake_format/pypi/setup.py#L55

Thanks!

**Comment by cheshirekow** [comment]
You are correct, I forgot to add the dependency to setup.py. Thanks for the report!

**Comment by cheshirekow** [comment]
Fixed in `v0.6.7`

**Comment by sergeyklay** [comment]
@cheshirekow This fix is conflicts with `conan` now. After updating cmake-format I got following error:

```
ERROR: conan 1.21.1 has requirement six<1.13.0,>=1.10.0, but you'll have six 1.14.0 which is incompatible.
```

Is there any real reason to use use highest version of `six`?
https://github.com/cheshirekow/cmake_format/blob/752d0453ec4a349f26c36523a1093aa8798342fb/cmake_format/pypi/setup.py#L56
